gulpfile.js

```diff
@@ -104,6 +104,7 @@ declareExtension('amp-selector', '0.1', true);
  */
 declareExtension('amp-slides', '0.1', false, 'NO_TYPE_CHECK');
 declareExtension('amp-social-share', '0.1', true);
+declareExtension('amp-tabs', '0.1', true);
```

Comment:
test is failing because it is looking for a CSS file. Can either make this `false` or just add a skeleton css file (amp-tabs will need a css file anyway)